src/games/avalon/views/game/goOnMission.tsx

```diff
@@ -163,32 +164,42 @@ export const GoOnMission: React.SFC<{ mission: Mission }> = ({ mission }) => {
     return (
       <>
         <Typography gutterBottom variant="h2">
-          {!allCardsPlayed && 'Waiting for the others to play their cards'}
+          {!allCardsPlayed &&
+            <FullScreenNotice>
+              Waiting for the others to play their cards
+            </FullScreenNotice>
+          }
           {game.excalibur && !mission.hasSwitched && 'Waiting on excalibur to decide'}
-          {allCardsPlayed && (!game.excalibur || mission.hasSwitched) && 'Waiting on you to flip votes'}
+          {allCardsPlayed && (!game.excalibur || mission.hasSwitched) 
+          && 
+            <FullScreenNotice clickHandler={flipCards}>
+              Flip Votes
+            </FullScreenNotice>
+          }
         </Typography>{' '}
-        <ActionRow fixed>
-          <Button confirm onClick={cancel}>
-            cancel
-          </Button>
-          <Button disabled={!allCardsPlayed} onClick={flipCards}>
-            flip cards
-          </Button>
-        </ActionRow>
       </>
     )
   }
 
   if (hasPlayedCard) {
     return (
       <Typography gutterBottom variant="h2">
-        {!allCardsPlayed && 'Waiting for the others to play their cards'}
+        {!allCardsPlayed && 
+          <FullScreenNotice>
+            Waiting for the others to play their cards
+          </FullScreenNotice>
+        }
         {game.excalibur &&
           !mission.hasSwitched &&
           'Waiting on excalibur to decide to flip a vote'}
         {allCardsPlayed &&
           (!game.excalibur || mission.hasSwitched) &&
-          'Waiting on owner to flip cards'}
+          'Waiting on owner to flip cards'
```

Comment:
We can get rid of this line since we have the notice down bellow

Reply:
Done.